org.sonatype.m2e.webby/src/org/sonatype/m2e/webby/internal/util/PathSelector.java

```diff
@@ -57,9 +57,8 @@ private static String normalizePattern(String pattern) {
     return normalized;
   }
 
-  private static String normalizePath(String path) {
-    String normalized = path.replace((File.separatorChar == '/') ? '\\' : '/', File.separatorChar);
-    return normalized;
+  public static String normalizePath(String path) {
```

Comment:
I have try this on windows and my test WarConfigurationExtractorTest is in error : 
````
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 1
	at java.lang.String.charAt(Unknown Source)
	at java.util.regex.Matcher.appendReplacement(Unknown Source)
	at java.util.regex.Matcher.replaceAll(Unknown Source)
	at java.lang.String.replaceAll(Unknown Source)
	at org.sonatype.m2e.webby.internal.util.PathSelector.normalizePath(PathSelector.java:61)
	at org.sonatype.m2e.webby.internal.config.WarConfigurationExtractor.resolve(WarConfigurationExtractor.java:202)
 ````